src/datasets/transformations.jl

```diff
@@ -41,8 +42,9 @@ struct NamedTupleData{TData, F}
     namedfs::NamedTuple{F}
 end
 
-LearnBase.nobs(data::NamedTupleData) = nobs(getfield(data, :data))
+# LearnBase functions {nobs, getobs} for NamedTupleData
```

Comment:
These changes are not related to this PR. We like to separate changes into different PRs when possible. Can you move these to another PR?

Reply:
Will do.

Reply:
I think separating out these changes is still outstanding